### DISCUSSION TITLE
Conditional resolver?

### DISCUSSION THREAD

**Discussion by jbockle** [ChilliCream/hotchocolate#5620] [Q&A]
Is there a way to block a field resolver when an object type is obtained from a mutation vs a query? My backend is eventually consistent and requesting a field that was just mutated would likely not have the up to date value.

**Comment by tobias-tengler** [comment]
What do you mean by blocking the resolver? Returning a default value? Omitting the field from the response? 
The first option would suggest an invalid value to your consumers. The second option breaks the spec.
Can't you simply omit the fields in question from your selection set for the mutation result?
